watertap/unit_models/zero_order/feed_zo.py

```diff
@@ -39,6 +42,37 @@ class FeedZOData(FeedData):
     def build(self):
         super().build()
 
+        units = self.config.property_package.get_metadata().get_derived_units
```

Comment:
I always imagined that we would just use the IDAES feedblock and then use the calculate_state method that I added to WaterTAP property packages so that users explicitly call a calculation to fix the state variables given the flowrate and concentrations. But I see what you did here and that works too.